### ISSUE TITLE
About the performance

### ISSUE THREAD

**Issue by Pefect96** [navervision/lincir#11]
I tried to reproduce the whole training, but found that the performance did not reach the level shown in the paper, and it would not increase after about 5000 steps, and it might decline in the following iteration, and the performance improvement was not obvious compared with the beginning, may I ask if the learning rate has been maintained at 1e-4? Whether adjusting the learning rate can further improve performance.

**Comment by SanghyukChun** [comment]
Hi, I usually used a fixed LR when I tested LinCIR with different training datasets. This is because LinCIR only has one small MLP module to be trained (the others are fixed). According to my previous experiences in other fields, too much fine LR tuning may cause overfitting; hence I usually use a fixed LR. Also, adjusting LR introduces another hyperparameter.
In my opinion, the performance upper bound is limited by the choice training dataset (as shown in Tab 9 of our paper), rather than optimization hyperparameters. If you still suffer from the reproducibility problem, please let me know.
